git-clang-format

```diff
@@ -577,4 +577,4 @@ def convert_string(bytes_input):
         return str(bytes_input)
 
 if __name__ == '__main__':
-  main()
+  main()
```

Comment:
Nit, there's still a change to this file.

Reply:
I will get this cleaned up..thank you

Reply:
Updated now .. added missing new line at end of file which is corrected